checklists/alz_checklist.en.json

```diff
@@ -2019,6 +2019,16 @@
             "training": "https://azure.github.io/azure-monitor-baseline-alerts/patterns/alz/",
             "link": "https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/landing-zone/design-area/management-monitor"
         },
+        {
+            "category": "Management",
+            "subcategory": "Monitoring",
+            "text": "Export logs to Azure Storage if your log retention requirements exceed seven years. Use immutable storage with a write-once, read-many policy to make data non-erasable and non-modifiable for a user-specified interval.",
```

Comment:
@mbilalamjad - This seems to be a duplicate with  "G03.02", should we look to consolidate?

Reply:
Good catch let me remove this one and lets keep G03.02 for now